modules/google-anyscale-vpc-firewall/main.tf

```diff
@@ -91,7 +91,7 @@ resource "google_compute_network_firewall_policy_rule" "ingress_allow_from_cidr_
   count   = local.ingress_from_cidr_blocks_enabled ? length(var.ingress_from_cidr_map) : 0
   project = var.anyscale_project_id
 
-  description     = lookup(var.ingress_from_cidr_map[count.index], "description", "Ingress Rule")
+  description     = try(var.predefined_firewall_rules[lookup(var.ingress_from_cidr_map[count.index], "rule", "")].description, "Ingress Rule")
```

Comment:
This is too long which makes it hard to understand...
How about use some locals to help?

```
locals {
  ingress_value = lookup(var.ingress_from_cidr_map[count.index], "rule", null)
  predefined_firewall_rule = local.ingress_value != null? var.predefined_firewall_rules[local.ingress_value]): null
}

description = local.predefined_firewall_rule != null? local.predefined_firewall_rule.description: "Ingress Rule"
```

Reply:
The trick here is that it's in a count loop—there could be 0, there could be 10, and each rule could (does) have a different description. I don't know how locals would be simpler in that situation.

Reply:
Keeping as is for now - will add a note to re-evaluate if we have a better way to handle this entire resource.